ONSdigital/census-rm-fieldwork-adapter
# Remove response received from action cancel

# Motivation and Context
actually need to remove responses received from actionCancel

# What has changed
removed it
Also simplified the dtos received from caseApi

# How to test?
with the related case-api branch should work with master ATs
https://github.com/ONSdigital/census-rm-case-api/pull/24

Check in rabbitmq that the queue RM.Field has some messages where the actionCancel only has these fields.  There's no new ATs for this as it's kind of proving a negative
<actionCancel><caseId>ee31238e-708a-46b0-98ad-03dda7c278eb</caseId><addressType>HH</addressType></actionCancel>



# Links
https://trello.com/c/B7P71yDJ/1042-cancel-refusal-message-to-the-field-is-incorrect
# Screenshots (if appropriate):
